implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -67,65 +67,25 @@ private Converters() {
 
     static final Converter<ConfigValue> CONFIG_VALUE_CONVERTER = new ConfigValueConverter();
 
-    static final Converter<String> STRING_CONVERTER = BuiltInConverter.of(0, newEmptyValueConverter(value -> value));
```

Comment:
I've made a similar proposal in https://github.com/smallrye/smallrye-config/pull/1249, but we ended up not moving forward with it.